docs/source/configurator.rst

```diff
@@ -125,6 +125,9 @@ Node type dependent options for [nodeN] : ::
 
    type="distributor" : No options
 
+   type="denester"
+   └─sequence_identifier : sequence identifier, default "Denester1"
```

Comment:
This option is not (yet) supported by the Denester.

Reply:
Again, to be fixed by pushing more commits here - we have this fixed in the BBC fork. Apologies!

Reply:
OK, no problem! So I will put my review on hold, until I get news from you regarding this MR.